dev/bots/test.dart

```diff
@@ -674,7 +674,8 @@ Future<void> _verifyVersion(String filename) async {
     exit(1);
   }
   final RegExp pattern = RegExp(r'^[0-9]+\.[0-9]+\.[0-9]+(-pre\.[0-9]+)?$');
-  if (!version.contains(pattern)) {
+  final RegExp hotfixPattern = RegExp(r'^v([0-9]+)\.([0-9]+)\.([0-9]+)(?:\+hotfix\.([0-9]+))?-([0-9]+)-g([a-f0-9]+)$');
```

Comment:
Use `\d` for digits, add uppercase for hex digits, and you don' t need most of the groups, since you don't ever read the groups.
```suggestion
  final RegExp hotfixPattern = RegExp(r'^v\d+\.\d+\.\d+(-hotfix\.\d+)?(-\d+)?(-g[a-fA-F0-9]+)?$');
```